nhost/compose/config.go

```diff
@@ -33,13 +30,14 @@ const (
 	// --
 
 	// container ports
-	graphqlPort = 8080
-	dbPort      = 5432
-	proxyPort   = 1337
+	graphqlPort  = 8080
+	dbPort       = 5432
+	proxyPort    = 1337
+	proxySSLPort = 443
 	// --
 
 	// default docker images
-	svcDashboardDefaultImage = "nhost/dashboard:0.9.10"
+	svcDashboardDefaultImage = "dashboard" // TODO: update image name before release
```

Comment:
# IMPORTANT: update this value before merging
This is set to `dashboard` for easier local testing with changes from https://github.com/nhost/nhost/pull/1467

Build docker image with changes from the mentioned PR above with: `docker build -f dashboard/Dockerfile -t dashboard . `